legacy/src/DataProvider.js

```diff
@@ -109,6 +109,6 @@ export class DiskDataProvider extends DataProvider {
   }
 
   nodeExists (name) {
-    return this.nodes[name].discovery !== null;
+    return this.nodes[name].discovery != null;
```

Comment:
Why make this change?

Reply:
`x !== null` checks whether the value is not `null`, `x != null` checks whether the value is not `null` or `undefined`. In the other dataset `.discovery` is not defined thus undefined.